stdlib/string.mc

```diff
@@ -159,50 +197,7 @@ recursive
          else concat (concat (head strs) delim) (strJoin delim (tail strs))
 end
 
-mexpr
-
-utest string2int "5" with 5 in
-utest string2int "25" with 25 in
-utest string2int "314159" with 314159 in
-utest string2int "-314159" with (negi 314159) in
-
-utest int2string 5 with "5" in
-utest int2string 25 with "25" in
-utest int2string 314159 with "314159" in
-utest int2string (negi 314159) with "-314159" in
-
-utest float2string 5.0e+25 with "5.0e+25" in
-utest float2string (negf 5.0e+25) with "-5.0e+25" in
-utest float2string (5.0e-5) with "5.0e-5" in
-utest float2string (negf 5.0e-5) with "-5.0e-5" in
-
-utest strIndex '%' "a % 5" with Some(2) in
-utest strIndex '%' "a & 5" with None () in
-utest strIndex 'w' "Hello, world!" with Some(7) in
-utest strIndex 'w' "Hello, World!" with None () in
-utest strIndex 'o' "Hello, world!" with Some(4) in
-utest strIndex '@' "Some @TAG@" with Some(5) in
-
-utest strLastIndex '%' "a % 5" with Some(2) in
-utest strLastIndex '%' "a & 5" with None () in
-utest strLastIndex 'w' "Hello, world!" with Some(7) in
-utest strLastIndex 'w' "Hello, World!" with None () in
-utest strLastIndex 'o' "Hello, world!" with Some(8) in
-utest strLastIndex '@' "Some @TAG@" with Some(9) in
-
-utest strSplit "ll" "Hello" with ["He", "o"] in
-utest strSplit "ll" "Hellallllo" with ["He", "a", "", "o"] in
-utest strSplit "" "Hello" with ["Hello"] in
-utest strSplit "lla" "Hello" with ["Hello"] in
-utest strSplit "Hello" "Hello" with ["", ""] in
-
-utest strTrim " aaaa   " with "aaaa" in
-utest strTrim "   bbbbb  bbb " with "bbbbb  bbb" in
-utest strTrim "ccccc c\t   \n" with "ccccc c" in
-
-utest strJoin "--" ["water", "tea", "coffee"] with "water--tea--coffee" in
-utest strJoin "--" [] with "" in
-utest strJoin "--" ["coffee"] with "coffee" in
-utest strJoin "water" ["coffee", "tea"] with "coffeewatertea" in
-
-()
+utest strJoin "--" ["water", "tea", "coffee"] with "water--tea--coffee"
+utest strJoin "--" [] with ""
+utest strJoin "--" ["coffee"] with "coffee"
+utest strJoin "water" ["coffee", "tea"] with "coffeewatertea"
```

Comment:
Like! A possible downside with the changes to the stdlib files is that it adds a kind of inconsistency of utest usage in stdlib (top level/not top level).